llm_on_ray/inference/serve.py

```diff
@@ -131,6 +131,11 @@ def main(argv=None):
     parser.add_argument(
         "--max_batch_size", default=None, type=int, help="The max batch size for dynamic batching."
     )
+    parser.add_argument(
+        "--new_deployment",
+        action="store_true",
+        help="Whether to override the previous deployment.",
+    )
```

Comment:
Maybe we can add a parameter like "openai_route_prefix"(default is "/"), If users don't want to kill the previous deployments of multiple models, they can set a new route_prefix by this parameter for new added models.

Reply:
Yes this is a good and simpler solution, will amend the PR